### PR TITLE
feat(terraform): add new dockerfile for terraform 0.12.0

### PR DESCRIPTION
**Pull request by julienpierini**

### CODE REVIEW

**Comment by Mallear** [comment]
LGTM
@mremi Can you check it an merge please ?

File: terraform/dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.6-alpine
+
+LABEL maintainer="Julien Pierini <julien.pierini@ekino.com>"
+
+ARG TERRAFORM_VERSION
+
+RUN apk update
```
Comment (mremi) [inline terraform/dockerfile]: Please merge the `RUN` commands to have only one

Comment (mremi) [reply terraform/dockerfile]: Maybe we should install the [ci-helper](https://github.com/rande/gitlab-ci-helper) as well like other images?

Comment (Mallear) [reply terraform/dockerfile]: Ci-helper isn't necessary in this image. We do not have to manage build dependencies when deploying with Terraform.

**Comment by mremi** [comment]
Why don't use [hashicorp/terraform](https://hub.docker.com/r/hashicorp/terraform)?

**Comment by JekPi** [comment]
because hashicorp/terraform doesn't allow other command than terraform itself (entrypoint on terraform binnary)

**Comment by mremi** [comment]
You can override the default entrypoint through `docker run --entrypoint` or in a docker compose file (see https://docs.docker.com/compose/compose-file/#entrypoint)

**Comment by JekPi** [comment]
Yes but we need python installed in our image and it's easier for us ton start from python-alpine than terraform image

File: terraform/dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.6-alpine
+
+LABEL maintainer="Julien Pierini <julien.pierini@ekino.com>"
+
+ARG TERRAFORM_VERSION
+
+RUN apk update \
+    && apk add bash unzip wget ca-certificates \
```
Comment (mremi) [inline terraform/dockerfile]: please sort by name

**Pull request merged**